build.gradle.kts

```diff
@@ -21,7 +21,7 @@ buildscript {
 }
 
 plugins {
-    kotlin("jvm") version "1.3.50"
+    kotlin("jvm") version "1.3.60"
```

Comment:
Maybe update this in `README.md` as well

Reply:
Okay, I will!